src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,89 @@
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage?: number;
+  currentPage?: number;
```

Comment:
Consider making `currentPage` a required prop if it is always expected to be provided by the parent component. Otherwise, ensure that the default value of 1 is appropriate for all use cases.